### DISCUSSION TITLE
User 닉네임 Validation에 관하여

### DISCUSSION THREAD

**Discussion by Samuel-0930** [FRONTENDSCHOOL8/Book-Kong#111] [Ideas]
회원가입할 때 user의 닉네임에 특수문자, 공백 등을 포함해도 되는지 얘기해볼 필요가 있을 것 같습니다. 저는 특수문자는 포함하면 안되고 닉네임 앞과 뒤의 공백은 자동으로 trim하고 닉네임 중간의 하나 이상의 공백은 하나로 자동으로 replace되는게 좋을 것 같습니다! 여러분은 어떠신지요~

### 예시 
![image](https://github.com/FRONTENDSCHOOL8/Book-Kong/assets/74633959/055e19ba-5421-4686-bc62-c3942f418a7b)

**Comment by DragonTiger92** [comment]
저도 동의합니다! 근데 이거 정규식으로 아예 처음에는 공백 못 치게 막는 건 못 하는 건가요??????

**Comment by yen815** [comment]
저도 동의합니다!

**Comment by DragonTiger92** [comment]
이거 지금 더 논의할 필요 없을 것 같은데, discussion 닫아도 괜찮을까요???? @Samuel-0930 @yen815 @juhyuny @HHeoGeon

**Reply by yen815** [reply]
넹

**Reply by DragonTiger92** [reply]
의견이 없어서, 이 디스커션은 지금 더 논의할 필요 없을 것 같으니까 일단 닫을게. 나중에 논의 필요하면 다시 열자!

**Comment by DragonTiger92** [comment]
현재 논의할 사항들 중에서 우선순위가 낮아 일단 discussion 닫기로 함.
